addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -833,9 +871,33 @@ public void run() {
             if (getThing().getStatus() == ThingStatus.ONLINE) {
                 if (isAwake()) {
                     requestData(TESLA_CHARGE_STATE);
+                    try {
+                        Thread.sleep(1000);
```

Comment:
This does not look like a very elegant solution... Is this something to stay or rather a quick hack trying to get around some API timing issues?